examples/embed-map/embed-map.js

```diff
@@ -0,0 +1,17 @@
+function initMap(el, options){
+  const map = new google.maps.Map(el, {
+    center: options.cityCoordinates,
+    zoom: 12
+  });
+
+  new google.maps.Marker({
+    position: options.coordinates,
+    map: map
+  });
+}
+
+module.exports = function embedMap(el, props) {
+  google.maps.event.addDomListener(window, 'load', function onLoad () {
+    initMap(el, props);
```

Comment:
Do we need to attached this to an event listener? I would have thought it’d work fine without it?